### PR TITLE
Enable access logging in Riak-CS

### PR DESCRIPTION
**Pull request by kellymclaughlin**

### CODE REVIEW

**Comment by kellymclaughlin** [comment]
This is ready for review in conjunction with [this](https://github.com/basho/webmachine/pull/108) Webmachine pr. The `app.config` template has been changed so that access logging is enabled by default. It can be disabled by commenting out the `webmachine_log_handler` entry and there is also instructions on how to dynamically add and remove log handlers in the wiki page that is referenced in the Webmachine pr.

**Comment by reiddraper** [comment]
Could be my setup (so I'm still investigating), but I'm getting this error with the WM branch.

```
(riak-cs@127.0.0.1)1> 12:46:32.950 [error] gen_event riak_cs_access_log_handler installed in webmachine_log_event terminated with reason: {noreply,{state,3600,1000000,0,{{{2013,2,11},{17,46,29}},{{2013,2,11},{18,0,0}}},#Ref<0.0.0.125>,32795}}
12:46:32.957 [error] gen_event riak_cs_access_log_handler installed in webmachine_log_event terminated with reason: {noreply,{state,3600,1000000,0,{{{2013,2,11},{17,46,32}},{{2013,2,11},{18,0,0}}},#Ref<0.0.0.857>,122938}}
12:46:32.976 [info] other
12:46:32.988 [error] gen_event riak_cs_access_log_handler installed in webmachine_log_event terminated with reason: {noreply,{state,3600,1000000,0,{{{2013,2,11},{17,46,32}},{{2013,2,11},{18,0,0}}},#Ref<0.0.0.873>,127035}}
12:46:33.021 [error] gen_event riak_cs_access_log_handler installed in webmachine_log_event terminated with reason: {noreply,{state,3600,1000000,1,{{{2013,2,11},{17,46,32}},{{2013,2,11},{18,0,0}}},#Ref<0.0.0.895>,131132}}

(riak-cs@127.0.0.1)1> 
```

File: src/riak_cs_access_log_handler.erl [hunk 1]
```diff
@@ -129,121 +160,39 @@ start_link(_BaseDir) ->
...
                     %% still a "small" log; keep going
-                    {noreply, State#state{size=S+1}};
+                    {ok, State#state{size=S+1}};
                 false ->
                     %% log is now "big"; flush it
-                    {noreply, force_archive(State, calendar:universal_time())}
+                    {ok, force_archive(State, calendar:universal_time())}
             end;
         _ ->
             {noreply, State}
```
Comment (reiddraper) [inline src/riak_cs_access_log_handler.erl]: think this should be `{ok, State}`, not `noreply`.

**Comment by kellymclaughlin** [comment]
Fixed `noreply` issue with [bcc72b1](https://github.com/basho/riak_cs/commit/bcc72b186acf686eacfd477e6ef2efcf736d8da9)

**Comment by reiddraper** [comment]
+1

**Pull request merged**